discovery-provider/src/schemas/schema_track.json

```diff
@@ -0,0 +1,291 @@
+{
```

Comment:
nit, i would call this
/schemas/track_schema.json
or just
/schemas/track.json